src/custom/state/price/updater.ts

```diff
@@ -236,43 +237,28 @@ export default function FeesUpdater(): null {
     if (!unsupportedToken && lastUnsupportedCheck) {
       setLastUnsupportedCheck(null)
     }
-
-    // Callback to re-fetch both the fee and the price
-    const refetchQuoteIfRequired = () => {
-      // IS an unsupported token and it's been greater than the threshold time
-      const unsupportedNeedsCheck = unsupportedTokenNeedsRecheck(unsupportedToken, lastUnsupportedCheck)
-
-      // if no token is unsupported and needs refetching
-      const hasToRefetch = !unsupportedToken && isRefetchQuoteRequired(isLoading, quoteParams, quoteInfo)
-
-      if (unsupportedNeedsCheck || hasToRefetch) {
-        // Decide if this is a new quote, or just a refresh
-        const thereIsPreviousPrice = !!quoteInfo?.price?.amount
-        const isPriceRefresh = quoteInfo
-          ? thereIsPreviousPrice && quoteUsingSameParameters(quoteParams, quoteInfo)
-          : false
-
-        setLastUnsupportedCheck(Date.now())
-
-        refetchQuote({
-          quoteParams,
-          fetchFee: true, // TODO: Review this, because probably now doesn't make any sense to not query the feee in some situations. Actually the endpoint will change to one that returns fee and quote together
-          previousFee: quoteInfo?.fee,
-          isPriceRefresh,
-        }).catch((error) => console.error('Error re-fetching the quote', error))
-      }
+    // IS an unsupported token and it's been greater than the threshold time
+    const unsupportedNeedsCheck = unsupportedTokenNeedsRecheck(unsupportedToken, lastUnsupportedCheck)
+
+    // if no token is unsupported and needs refetching
+    const hasToRefetch = !unsupportedToken && isRefetchQuoteRequired(isLoading, quoteParams, quoteInfo)
+
+    if (unsupportedNeedsCheck || hasToRefetch) {
+      // Decide if this is a new quote, or just a refresh
+      const thereIsPreviousPrice = !!quoteInfo?.price?.amount
+      const isPriceRefresh = quoteInfo
+        ? thereIsPreviousPrice && quoteUsingSameParameters(quoteParams, quoteInfo)
+        : false
+
+      setLastUnsupportedCheck(Date.now())
+
+      refetchQuote({
+        quoteParams,
+        fetchFee: true, // TODO: Review this, because probably now doesn't make any sense to not query the feee in some situations. Actually the endpoint will change to one that returns fee and quote together
```

Comment:
I guess this no longer applies.
But no need to change it in this PR as the change is unrelated